File: cs50x/aula_c/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    //Get size of grid
    //const int n = 3;
    //int n = get_int("Size: ");*/
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1);

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: cs50x/pset6/mario/more/mario.py
from cs50 import get_int

# height in range 1 to 8, inclusive
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# print pyramid
for i in range(height):
    print(" " * (height - 1 - i), "#" * (i + 1), "  ", "#" * (i + 1), sep="")